ui/src/main/js/page/channel/jira/cloud/JiraCloudDistributionConfiguration.js

```diff
@@ -85,6 +85,17 @@ const JiraCloudDistributionConfiguration = ({
                 errorName={FieldModelUtilities.createFieldModelErrorKey(JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.reopenWorkflow)}
                 errorValue={errors.fieldErrors[JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.reopenWorkflow]}
             />
+            <TextInput
+                id={JIRA_CLOUD_DISTRIBUTION_FIELD_KEYS.issueSummary}
+                label="Issue Summary"
+                description="The summary to use for each issue created. Can use the following variables to populate data from the message content {{providerName}}, {{projectName}}, {{projectVersion}}, {{componentName}}, {{componentVersion}}, {{severity}}"
```

Comment:
We have these documented in confluence currently for the Jira custom fields. As we need to add more and more replacement values custom fields we would need to change this description each time as well. Perhaps we need a new section in confluence to track these changes and direct customers there? https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/520781913/Configuring+Distribution+Jobs+in+Alert#Jira-Server

Reply:
Yes this is definitely something that we should be concerned about especially as we add more fields as you suggested. I will add an update to the documentation for this and perhaps as we add more, we remove them from the description when it gets too long.